plugins/woocommerce-blocks/packages/components/text-input/style.scss

```diff
@@ -44,7 +44,7 @@
 	input[type="number"],
 	input[type="email"] {
 		@include font-size(regular);
-		padding: em($gap);
+		padding: em($gap) em($gap-smaller);
```

Comment:
This padding change and the max-height change below was to prevent layout shift when focussing on the input. It either pushed the "apply" button downwards or to the right depending on wrapping.